client/src/components/History/CurrentCollection/CollectionDetails.vue

```diff
@@ -8,6 +8,7 @@
         <template v-slot:name>
             <h3 v-short="dsc.name || 'Collection'" data-description="collection name display" />
             <CollectionDescription
+                :collection="dsc"
                 :collection-type="dsc.collection_type"
```

Comment:
It looks like care was taken here to unpack `dsc` into appropriate props instead of passing it all the way through; do you mind refactoring this to send the jobStateSummary in explicitly?